docs/EVG.md

```diff
@@ -750,6 +755,18 @@ Before issuing an EV Certificate, the CA MUST ensure that all Subject organizati
 
 As a general rule, the CA is responsible for taking all verification steps reasonably necessary to satisfy each of the Verification Requirements set forth in the subsections below.  The Acceptable Methods of Verification set forth in each of Sections 11.2 through 11.14 (which usually include alternatives) are considered to be the minimum acceptable level of verification required of the CA.  In all cases, however, the CA is responsible for taking any additional verification steps that may be reasonably necessary under the circumstances to satisfy the applicable Verification Requirement.
 
+### 11.1.3.  Disclosure of Verification Sources
+
+Effective as of 1 October 2020, prior to the use of an Incorporating Agency or Registration Agency to fulfill these verification requirements, the CA MUST publicly disclose, through an appropriate and readily accessible online means, Agency Information about the Incorporating Agency or Registration Agency.
+
+This Agency Information SHALL include at least the following:
+* Sufficient information to unambiguously identify the Incorporating Agency or Registration Agency (such as a name, jurisdiction, and website); and,
+* The accepted values for the `subject:jurisdictionLocalityName` (OID: 1.3.6.1.4.1.311.60.2.1.1), `subject:jurisdictionStateOrProvinceName` (OID: 1.3.6.1.4.1.311.60.2.1.2), and `subject:jursidictionCountryName` (OID: 1.3.6.1.4.1.311.60.2.1.3) fields when a certificate is issued using information from that Incorporating Agency or Registration Agency, indicating the jurisidction(s) that the Agency is appropriate for; and,
```

Comment:
It is not clear here whether values (plural) refers collectively to JLN, JSOPN, and JCN, each of which has a unique value, or whether it's permitted to have multiple acceptable values for one or more of them.  The reason this gets interesting is because there are often information sources that are aggregations of other information sources, so one information source may cover multiple registration localities (that may not publish their own information sources).

Reply:
Since this is solely about disclosure/transparency, I’m totally fine taking a very liberal approach. I’ll open up an in-line suggestion